lib/strategies/simple.js

```diff
@@ -76,10 +92,16 @@ class SimpleConcat {
     }
 
     let entry = {
```

Comment:
My early thoughts were something like:

```js
class Entry {
  constructor(path) {
    this._content = undefined;
   this.path = path;
  }

  get  content() {
     if (this._content !== undefined) { return this._content; }
     let result = fs.readFileSync(this.path, 'UTF8');
     if (result.length < TRESHOLD) { this._content = content; }

     return result;
  }
}
```

Obviously I am open to other approaches. Just figured I would share.

Reply:
So, if I'm understanding this right, in this approach, we will not be updating the content in entry during add and update the first time. It would be computed just in result.
I think this might make the PR a lot simpler than what I have written and also get rid of the global baseDir(I really didn't want to do that).

Reply:
However, it seems like doing this might complicate writing unit tests as content need to be read instead of being assigned directly

Reply:
Thoughts?

Reply:
> However, it seems like doing this might complicate writing unit tests as content need to be read from file instead of being assigned directly

Slightly more yes, but using [node-fixturify](https://github.com/joliss/node-fixturify) should nearly mitigate this issue. (I think)

Reply:
I have made changes as per the suggested approach. This does look cleaner. But, I'm not sure about the changes I have made to the unit tests.

If these changes look good, I can squash the commits together